### PR TITLE
db/climate-map

### PR DESCRIPTION
**Pull request by bre1470**

### CODE REVIEW

File: samples/dashboard/demos/climate/demo.js [hunk 1]
```diff
@@ -144,32 +153,32 @@ async function buildDashboard() {
                         }
                     },
                     marker: {
-                        lineColor: '#FFF',
-                        lineWidth: 1,
+                        enabled: true,
                         radius: 6,
+                        // symbol: 'mapmarker'
```
Comment (karolkolodziej) [inline samples/dashboard/demos/climate/demo.js:158]: Remove?

File: samples/dashboard/demos/climate/demo.js [hunk 2]
```diff
@@ -144,32 +153,32 @@ async function buildDashboard() {
...
                     },
                     tooltip: {
-                        footerFormat: void 0,
-                        headerFormat: void 0,
+                        // distance: 6, // mapmarker fix
```
Comment (karolkolodziej) [inline samples/dashboard/demos/climate/demo.js:161]: Remove?

**Review by karolkolodziej — APPROVED** [review]
There are still some functionalities missing: 

- [ ] selects a city (for now let's increase the pin/marker size)
- [ ] changes DataGrid component to display data about this city

Other issues are connected to the KPI and Navigator.
If you feel we should merge this, move on and revisit this issue after KPI and Navigator I'm fine with that.

File: samples/dashboard/demos/climate/demo.js [hunk 3]
```diff
@@ -76,45 +78,52 @@ async function buildDashboard() {
...
+
+                            // force point colors
+                            for (const point of this.series[1].points) {
+                                if (point.graphic && point.color) {
+                                    point.graphic.element.style.fill =
+                                        point.color;
+                                }
+                            }
+
+                            console.log('redraw', this.series[1].points);
```
Comment (karolkolodziej) [inline samples/dashboard/demos/climate/demo.js:96]: Unnecessary console log- maybe at that point we should enable the `eslint`?

**Review by karolkolodziej — APPROVED** [review]

File: samples/dashboard/demos/climate/demo.js (hunk 1 above)
Comment (bre1470) [reply samples/dashboard/demos/climate/demo.js:158]: Not yet, mapmarker needs to be fixed in separate task.

File: samples/dashboard/demos/climate/demo.js (hunk 3 above)
Comment (bre1470) [reply samples/dashboard/demos/climate/demo.js:96]: Yes, true. I will activate it.

**Comment by bre1470** [comment]
@karolkolodziej 
I forgot the first task. Regarding the second one I like to wait until we have the changes on datagrid component in place. Otherwise merge conflicts become messy and things have to be done twice.

**Comment by karolkolodziej** [comment]
Ok, so we're waiting for the first one to be fixed. 
Otherwise, it looks good ;)

**Review by sebastianbochan — APPROVED** [review]

**Review by karolkolodziej — APPROVED** [review]

**Comment by sebastianbochan** [comment]
I see that there is data for fog, ice, rain and temperature. How about other KPIs. Will you add it or should we reduce the KPIs?

**Comment by bre1470** [comment]
We have two temperatures, so we could continue with six KPI and have one KPI for date or lat/lon.

**Comment by sebastianbochan** [comment]
Will discuss it tomorrow ;)

**Pull request merged**